package.json

```diff
@@ -34,5 +38,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^6.1.0",
```

Comment:
I asked in the camp channel that you down grade this to 5.6.0, please do

Reply:
Yes, the mocking of server responses are not really up to speed in this version. Please downgrade.

Reply:
Bumped down to this version.